source/speech/speech.py

```diff
@@ -164,11 +164,13 @@ def processText(locale: str, text: str, symbolLevel: characterProcessing.SymbolL
 	return text.strip()
 
 
-def cancelSpeech():
+def cancelSpeech(clearBrailleRegions=True):

```

Comment:
Please always add type hints when making changes to function headers

```suggestion
def cancelSpeech(clearBrailleRegions: bool = True):
```